examples/tti/tti_example.py

```diff
@@ -23,10 +23,11 @@ def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
 
     model = IGrid(origin, spacing,
                   true_vp,
-                  .4*np.ones(dimensions),
-                  -.1*np.ones(dimensions),
-                  -np.pi/7*np.ones(dimensions),
-                  np.pi/5*np.ones(dimensions))
+                  rho=None,
```

Comment:
just a curiosity: I guess ``rho`` isn't here yet because the system is still unstable with the current operators?